### PR TITLE
Add unit tests for edu.princeton.cs.algs4.StdStats

### PR DESCRIPTION
**Pull request by EricHetti**
I've analysed your codebase and noticed that `edu.princeton.cs.algs4.StdStats` is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,12 +75,26 @@
...
 						</executions>
 					</plugin>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<version>2.22.1</version>
+						<configuration>
+							<useSystemClassLoader>false</useSystemClassLoader>
+						</configuration>
+					</plugin>
```
Comment (freddytuxworth) [inline pom.xml]: Do you need this explicit `surefire` config? Also, do you need the `useSystemClassLoader` setting? (AFAIK only required for mockito, powermock)

Comment (Braavos6) [reply pom.xml]: Removed

**Review by freddytuxworth — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -82,5 +82,11 @@
 		</profile>
 	</profiles>
 	<dependencies>
-	</dependencies>
```
Comment (petersear) [inline pom.xml:85]: Is there a white space difference here? Just wondering why `</dependencies>` thinks it's changed.

**Review by petersear — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (Braavos6) [reply pom.xml:85]: I am not sure...

**Pull request closed without merging**